src/qute/commands/registerCommands.ts

```diff
@@ -69,6 +70,56 @@ function registerJavaDefinitionCommand(context: ExtensionContext) {
   }));
 }
 
+/**
+ * Toggle for enabling/disabling Qute validation.
+ *
+ * @param context
+ */
+function registerQuteValidationToggleCommand(context: ExtensionContext) {
+  // Toggle on
+  context.subscriptions.push(commands.registerCommand(QuteSettings.QUTE_VALIDATION_ENABLED + QuteSettings.TOGGLE_ON, async (uri?: Uri) => {
+    const templateUri = uri.toString();
+    // from QuteTemplateValidationStatusCommandHandler
+    const result: TemplateValidationStatus = await commands.executeCommand(QuteServerCommandConstants.QUTE_VALIDATION_TEMPLATE_STATUS, templateUri);
+    if (!result.validationEnabled) {
+      // The current settings 'is 'qute.validation.enabled' is set to false, update it to true
+      const edit = {
+        scopeUri: templateUri,
+        value: true,
+        editType: ConfigurationItemEditType.Update,
+        section: QuteSettings.QUTE_VALIDATION_ENABLED
+      } as ConfigurationItemEdit;
+      await commands.executeCommand(QuteClientCommandConstants.COMMAND_CONFIGURATION_UPDATE, edit);
+    }
+    if (result.excluded.length > 0) {
+      // The current settings 'is 'qute.validation.excluded' contains at least one pattern
+      // Those patterns must be removed
+      const edit = {
+        scopeUri: templateUri,
+        value: result.excluded,
+        editType: ConfigurationItemEditType.Remove,
+        section: QuteSettings.QUTE_VALIDATION_EXCLUDED
+      } as ConfigurationItemEdit;
+      await commands.executeCommand(QuteClientCommandConstants.COMMAND_CONFIGURATION_UPDATE, edit);
+    }
+    await commands.executeCommand('setContext', 'quteValidationEnabled', true);
```

Comment:
So you have a file opened, and validation is on. You click the validation icon, the language server confirms it's enabled, and the exclusion list is empty. So none of the above code paths are taken, and then you execute this line, which just enables validation, so nothing happens.